### DISCUSSION TITLE
Entity Creation (spawnsent) Extension

### DISCUSSION THREAD

**Discussion by Denneisk** [wiremod/wire#2640] [Suggestions]
Create an extension that allows creating any entity. Why? Why not? Most likely going to be a heavily locked down extension anyway, may as well let those who want to.

**Comment by thegrb93** [comment]
I think adding new stuff goes in discussions, not issues.

**Comment by Denneisk** [comment]
With all the inane ideas I've been coming up with, I have to agree. ~~Will do that next time around.~~ Oh. didn't even need to do anything.
